xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/DXVAHD.cpp

```diff
@@ -17,6 +17,7 @@
 #include "VideoRenderers/RenderManager.h"
 #include "VideoRenderers/windows/RendererBase.h"
 #include "rendering/dx/RenderContext.h"
+#include "utils/StringUtils.h"
```

Comment:
This header is necessary?
It doesn't seem to make use of it, at least in the commit where it is "[Windows[dxva] promote ID3D11VideoProcessorEnumerator1 to class member"

Also typo in commit name `[Windows[dxva]`